src/SidebarTop.tsx

```diff
@@ -37,6 +37,11 @@ export const SidebarTop = ({ api }: SidebarTopProps) => {
   const [gitInfoError] = useSharedState<Error>(GIT_INFO_ERROR);
 
   const lastStep = useRef(localBuildProgress?.currentStep);
+  const { status } = useStorybookState();
+  const changedStoryCount = Object.values(status).filter(
+    (value) => value[ADDON_ID]?.status === "warn"
+  );
```

Comment:
It seems a bit iffy to pull the status update from SB's store like this. Would it be better to call `testsToStatusUpdate` on the local build?

Reply:
How is that different than what we are doing to set the value here? https://github.com/chromaui/addon-visual-tests/blob/77c3dfb859f5eaa1c0a7c964ff419ee0a3564a68/src/screens/VisualTests/VisualTests.tsx#L218-L230

Reply:
I guess I see your point.